ansible_builder/steps.py

```diff
@@ -37,14 +37,13 @@ def __init__(self, requirements_naming):
         """Assumes given requirements file name has been placed in the build context
         """
         self.steps = []
-        self.steps.append(
-            "ADD {0} /build/{0}".format(requirements_naming)
-        )
         self.steps.extend([
+            "ARG ANSIBLE_GALAXY_CLI_COLLECTION_OPTS=\"\"",
```

Comment:
Right now, this default of `""` can't be changed with `build_arg_defaults` in the execution environment definition, and maybe it arguably should be customizable there.